src/wombats/game/zakano_code.clj

```diff
@@ -379,15 +425,19 @@
     :validate-command (fn [] true)
     :equality-command (fn [prev next] prev)}])
 
-(defn calculate-next-command
+(defsource calculate-next-command
   [enriched-state]
-  (first (filter #(not (nil? %))
-                 (map #(xform-command enriched-state (:name %) (:fn %))
-                      command-priority))))
-
-(defn calculate-optimal-command
+  (first (filter (fn [command] (not (nil? command)))
```

Comment:
`some?` returns a boolean. Here I'm looking to get the first item in the command vector. So this got me looking into `some?` a little more. (in particular, `some?` vs `some`. `some?` will always return a boolean, `some` will return `true` or `nil`